### DISCUSSION TITLE
Spacy Context Understanding

### DISCUSSION THREAD

**Discussion by YasineNifa** [explosion/spaCy#5169] [Help: Other Questions]
I have a question please.
I tried to annotate this sentence "I am going to New York University"
In the first try, I write "University" with a capital letter and the spacy recognize the "New York University" as ORG
sentence1 = 'I study at New York University' => New York University ORG
In the second try. I write "I am going to New York" and spacy recognize "New York" as GPE
sentence2 = 'I am going to New York' => New York GPE.

I want to understand how spacy do recognize these entities ?
I want to know how the model learn that this one instance of “New York” in the first sentence is a Organisation and the second one is a Location. I understand that Spacy use the contexte of the word to predict the entity but how can it do that ?

**Comment by svlandeg** [comment]
spaCy uses Machine Learning models that learn how to make correct predictions by looking at training instances (https://spacy.io/usage/spacy-101#training and https://spacy.io/usage/training#ner).

A more detailed description of spaCy's NER model can be found in [this video by Matt](https://www.youtube.com/watch?v=sqDHBH9IjRU). It basically explains spaCy's transition-based approach to NER, which goes through the different words in the sentence and tries to predict the correct action (label) for each word that it encounters.

For generic questions about Natural Language Processing, Machine Learning and how to use spaCy in specific use-cases, you can probably find more help at a forum like StackOverflow. If you encounter specific bugs or problems, feel free to open a new issue here!
